apps/console/src/features/applications/components/application-list.tsx

```diff
@@ -352,7 +352,8 @@ export const ApplicationList: FunctionComponent<ApplicationListPropsInterface> =
             );
         }
 
-        if (list?.totalResults === 0) {
+        if (list?.totalResults === 0
+            || (list?.applications?.find(app => app.name === "wso2carbon-local-sp") && list?.totalResults === 1)) {
```

Comment:
Shall we get `wso2carbon-local-sp` to a constant. There is one more usage. Would be better to refactor that as well.

Reply:
Resolved by d6bf687